### PR TITLE
Add cc_emails attribute to account

### PR DESCRIPTION
**Pull request by benliscio**

### CODE REVIEW

**Comment by bhelx** [comment]
Thanks @benliscio

I thought we already added this but looks like we forgot. Will review this today.

**Comment by benliscio** [comment]
@bhelx FYI the docs do not list cc_email as an attribute for [account post](https://dev.recurly.com/docs/create-an-account) and [account put](https://dev.recurly.com/docs/update-account)

**Comment by bhelx** [comment]
@benliscio Thanks for pointing that out. I'm adding it into docs now.

**Comment by bhelx** [comment]
And thanks again for the contribution!

File: spec/fixtures/accounts/show-200.xml [hunk 1]
```diff
@@ -12,6 +12,7 @@ Content-Type: application/xml; charset=utf-8
   <account_code>abcdef1234567890</account_code>
   <username>shmohawk58</username>
   <email>larry.david@example.com</email>
+  <cc_emails>cheryl.hines@example.com,richard.lewis@example.com</cc_emails>
```
Comment (bhelx) [inline spec/fixtures/accounts/show-200.xml]: Clever :+:

**Pull request merged**